ridgeback_bringup/scripts/install

```diff
@@ -3,7 +3,29 @@
 import os
 import robot_upstart
 
-j = robot_upstart.Job(workspace_setup=os.environ['ROBOT_SETUP'])
-j.add(package="ridgeback_base", filename="launch/base.launch")
-j.add(package="ridgeback_bringup", filename="launch/accessories.launch")
-j.install()
+main_job = robot_upstart.Job(workspace_setup=os.environ['ROBOT_SETUP'],interface='br0')
+main_job.add(package="ridgeback_base", filename="launch/base.launch")
+main_job.add(package="ridgeback_bringup", filename="launch/accessories.launch")
+main_job.install()
+
+
+class CanBridgeProvider(robot_upstart.providers.Generic):
+    def generate_install(self):
+        return { "/etc/init/can-udp-bridge.conf": {
+            "content":  open(os.path.join(os.path.dirname(__file__), 'can-udp-bridge.conf'), 'r').read(),
```

Comment:
It's ugly having to put this file alongside the script. I'd suggest finding it using catkin.find_in_workspaces, eg:

https://github.com/clearpathrobotics/robot_upstart/blob/jade-devel/src/robot_upstart/providers.py#L137

It doesn't matter so much here since the script ends immediately, but open().read() is an antipattern. Consider instead:

```
with open(can_conf_file, "r") as f:
    can_conf_contents = f.read()

return { "/etc/init/can-udp-bridge.conf": {
             "content": can_conf_contents
             ...
```